winit-core/src/event_loop/pump_events.rs

```diff
@@ -1,8 +1,8 @@
 use std::time::Duration;
```

Comment:
Unsure if this place is the right spot for this? It somewhat matches `winit_core::application::macos` though.